rings: 556
min_radius: 1
interp: 1
full: 0
sectors: 556
sp: 0
min_disparity: -30
disparities: 80
sigma: 3.0
occ_likelihood: 0.1
pole: 0.6
spherical_angle_bins: 200
shell_radius: 1.0
high_pass: true
ego_frame: eyes_center_vision_link
left_camera_frame: l_camera_vision_link
right_camera_frame: r_camera_vision_link
uncertainty_lower_bound: 10000.0
uncertainty_upper_bound: 1000000.0
